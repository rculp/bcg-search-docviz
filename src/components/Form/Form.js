import { Form } from 'semantic-ui-react';
import './Form.scss';

export default Form;
