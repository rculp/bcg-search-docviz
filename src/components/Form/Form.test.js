import { shallow } from 'enzyme';

import React from 'react';

import Form from './Form';

describe('Form', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Form />
    );
    expect(component).toMatchSnapshot();
  });
});