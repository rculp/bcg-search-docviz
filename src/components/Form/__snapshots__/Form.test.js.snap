// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders and matches our snapshot 1`] = `
<form
  className="ui form"
  onSubmit={[Function]}
/>
`;
